SofaKernel/modules/Sofa.GL/Component/Engine/src/sofa/gl/component/engine/config.h.in

```diff
@@ -34,8 +34,8 @@
 // DEPRECATION MACROS
 // Keep the previous macros
 // This backward compatibility will be removed at v23.06
-#ifndef SOFAGENERALENGINE_HAVE_SOFA_GL
-#define SOFAGENERALENGINE_HAVE_SOFA_GL 1
+#ifndef SOFAGENERALENGINE_HAVE_SOFA_GL_COMPONENT_ENGINE
```

Comment:
Why the change ? These definitions should not be modified I think. (as they are used for compatibility)

Reply:
Wanted to be consistent with what is actually needed. I will re-add the compat :+1: